queries/mermaid/indents.scm

```diff
@@ -0,0 +1,13 @@
+[
+  (diagram_flow)
+  (flow_stmt_subgraph)
+] @indent.begin
+
+[
+  "end"
+] @indent.branch
```

Comment:
`@indent.end` seems to be missing here.

Also you need to run `scripts/format-queries queries/mermaid`

Reply:
> `@indent.end` seems to be missing here.

I'm not sure what you mean here. Is `@indent.end` a required part of an indent query in neovim (I didn't see that in the docs)? By making it branch it does the following

```{mermaid}
flowchart BT
  subgraph DA
    direction BT
    a --> b
  end
```
```mermaid
flowchart BT
  subgraph DA
    direction BT
    a --> b
  end
```
In terms of the "end" of the first indent. There appears to be no "end" in mermaid js, since I think its one chart per file. And the TS Query Tree is rooted at document. I can't find anything contrary to this with some cursory looking. (I mostly use it in markdown -- one chart per code block)

> Also you need to run `./scripts/format-queries.lua queries/mermaid`

I was waiting to do this until I knew that I had a general direction to get the pr going.